Projects/EditCoInsured/Sources/View/CoInsuredProcessingScreen.swift

```diff
@@ -57,12 +60,8 @@ struct CoInsuredProcessingScreen: View {
 
     private func missingContractAlert() {
         vm.store.send(.fetchContracts)
-        vm.store.send(.coInsuredNavigationAction(action: .dismissEditCoInsuredFlow))
-        DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
-            store.send(.checkForAlert)
```

Comment:
we are not calling this .checkForAlert anymore. Do we need to do anything about it?

Reply:
@sladan-hedvig it's checked elsewhere